src/server/game/Battlefield/Zones/BattlefieldTB.cpp

```diff
@@ -0,0 +1,839 @@
+                team = TEAM_ALLIANCE;
+                capturing = true;
+                break;
+            case BF_CAPTUREPOINT_OBJECTIVESTATE_ALLIANCE_HORDE_CHALLENGE:
+            case BF_CAPTUREPOINT_OBJECTIVESTATE_NEUTRAL_HORDE_CHALLENGE:
+                team = TEAM_HORDE;
+                capturing = true;
+                break;
+            default:
+                team = TEAM_NEUTRAL;
+        }
+
+        packet.Worldstates.emplace_back(uint32(TBCapturePoints[i].wsControlled[TEAM_ALLIANCE]), int32(team == TEAM_ALLIANCE && controlled ? 1 : 0));
+        packet.Worldstates.emplace_back(uint32(TBCapturePoints[i].wsCapturing[TEAM_ALLIANCE]), int32(team == TEAM_ALLIANCE && capturing ? 1 : 0));
+        packet.Worldstates.emplace_back(uint32(TBCapturePoints[i].wsNeutral), int32(team == TEAM_NEUTRAL ? 1 : 0));
+        packet.Worldstates.emplace_back(uint32(TBCapturePoints[i].wsCapturing[TEAM_HORDE]), int32(team == TEAM_HORDE && capturing ? 1 : 0));
+        packet.Worldstates.emplace_back(uint32(TBCapturePoints[i].wsControlled[TEAM_HORDE]), int32(team == TEAM_HORDE && controlled ? 1 : 0));
+    }
+
+    packet.Worldstates.emplace_back(uint32(TB_WS_VIGIL_ALLIANCE_CONTROLLED), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_VIGIL_ALLIANCE_CAPTURING), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_VIGIL_NEUTRAL), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_VIGIL_HORDE_CAPTURING), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_VIGIL_HORDE_CONTROLLED), int32(0));
+
+    packet.Worldstates.emplace_back(uint32(TB_WS_GARRISON_ALLIANCE_CONTROLLED), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_GARRISON_ALLIANCE_CAPTURING), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_GARRISON_NEUTRAL), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_GARRISON_HORDE_CAPTURING), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_GARRISON_HORDE_CONTROLLED), int32(0));
+
+    packet.Worldstates.emplace_back(uint32(TB_WS_TOWERS_DESTROYED_SHOW), int32(GetData(BATTLEFIELD_TB_DATA_TOWERS_DESTROYED)));
+    packet.Worldstates.emplace_back(uint32(TB_WS_BUILDINGS_CAPTURED_SHOW), int32(IsWarTime() ? 1 : 0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_BUILDINGS_CAPTURED), int32(GetData(BATTLEFIELD_TB_DATA_BUILDINGS_CAPTURED)));
+    packet.Worldstates.emplace_back(uint32(TB_WS_TOWERS_DESTROYED), int32(0));
+    
+    packet.Worldstates.emplace_back(uint32(TB_WS_TIME_BATTLE_END_SHOW), int32(IsWarTime() ? 1 : 0));
+
+    packet.Worldstates.emplace_back(uint32(TB_WS_STATE_BATTLE), int32(IsWarTime() ? 1 : 0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_STATE_PREPARATIONS), int32(GetState() == BATTLEFIELD_WARMUP ? 1 : 0));
+
+    packet.Worldstates.emplace_back(uint32(TB_WS_35_UNKNOWN), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_36_UNKNOWN), int32(0));
+    packet.Worldstates.emplace_back(uint32(TB_WS_37_UNKNOWN), int32(0));
```

Comment:
I couldn't find out, because their value is always zero. They are listed together in the smsg init world states packet when entering TB. I will comment them, just wanted to keep the same packet structure for now.